### PR TITLE
It books

### PR DESCRIPTION
**Pull request by nguyenhoanganhtuan1206**
2. Integrate Swagger

### CODE REVIEW

**Review by nguyenvanman — CHANGES_REQUESTED** [review]

**Comment by voquanghoa** [comment]
Bổ sung unit test

File: src/test/java/com/javatemplate/persistent/book/BookEntityMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+class BookEntityMapperTest {
+
+    @Test
+    void shouldToBook_Ok() {
+        final var bookEntity = buildBookEntity();
+        final var book = toBook(bookEntity);
+
+        assertEquals(bookEntity.getId(), book.getId());
+        assertEquals(bookEntity.getName(), bookEntity.getName());
```
Comment (nguyenvanman) [inline src/test/java/com/javatemplate/persistent/book/BookEntityMapperTest.java:19]: :notbad:

File: src/test/java/com/javatemplate/fakes/BookFakes.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import java.util.UUID;
+import java.util.stream.IntStream;
+
+@UtilityClass
+public class BookFakes {
+
+    public static Book buildBook() {
+        return Book.builder()
+                .id(UUID.randomUUID())
+                .name(RandomStringUtils.randomAlphabetic(3, 10))
```
Comment (nguyenvanman) [inline src/test/java/com/javatemplate/fakes/BookFakes.java:18]: Use static import

File: src/main/java/com/javatemplate/persistent/book/BookRepository.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.javatemplate.persistent.book;
+
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.UUID;
+
+public interface BookRepository extends CrudRepository<BookEntity, UUID> {
```
Comment (nguyenvanman) [inline src/main/java/com/javatemplate/persistent/book/BookRepository.java:7]:
```suggestion
@Repository
public interface BookRepository extends CrudRepository<BookEntity, UUID> {
```

File: src/main/java/com/javatemplate/domain/book/BookService.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class BookService {
+
+    private final BookStore bookStore;
+
+    public List<Book> findAll() {
+        final List<Book> books = bookStore.findAll();
```
Comment (nguyenvanman) [inline src/main/java/com/javatemplate/domain/book/BookService.java:18]:
```suggestion
        return bookStore.findAll();
```

**Review by nguyenvanman — CHANGES_REQUESTED** [review]

**Pull request merged**